### PR TITLE
Android - Super resolution sample

### PR DESCRIPTION
**Pull request by st-haingo**

### CODE REVIEW

**Comment by PaulTR** [comment]
Please change the gallery setting to work like iOS -> display the original image scaled to fit on the device screen, then allow users to drag their finger over sections to display a popup with the larger segment at run time.

File: examples/super_resolution/android/app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    implementation(libs.androidx.lifecycle.runtime.compose)
+    implementation(libs.androidx.lifecycle.viewmodel.compose)
+    implementation(libs.androidx.activity.compose)
+    implementation(platform(libs.androidx.compose.bom))
+    implementation(libs.androidx.ui)
+    implementation(libs.androidx.ui.graphics)
+    implementation(libs.androidx.ui.tooling.preview)
+    implementation(libs.androidx.material2)
+    implementation(libs.tensorflow.lite)
+    implementation(libs.tensorflow.lite.task.vision)
```
Comment (PaulTR) [inline examples/super_resolution/android/app/build.gradle.kts:71]: Task library isn't being used, so should be removed

**Comment by st-haingo** [comment]
> Please change the gallery setting to work like iOS -> display the original image scaled to fit on the device screen, then allow users to drag their finger over sections to display a popup with the larger segment at run time.

I updated it, please help me review it.
Thank you

**Comment by st-haingo** [comment]
Replace by https://github.com/PaulTR/interpreter-samples/pull/12

**Pull request closed without merging**